drafts/str4d-stratum/draft1.rst

```diff
@@ -0,0 +1,256 @@
+Abstract
+========
+
+This ZIP describes the Zcash variant of the Stratum protocol, used by miners to
+communicate with mining pool servers.
+
+
+Specification
+=============
+
+``mining.subscribe()``
+----------------------
+
+Request::
+
+    {"id": 1, "method": "mining.subscribe", "params": ["CONNECT_HOST", CONNECT_PORT, "MINER_USER_AGENT", "SESSION_ID"]}\n
+
+- ``CONNECT_HOST`` (str): The host string used by the miner.
+
+  - ``pool.example.com``
+
+- ``CONNECT_PORT`` (int): The port used by the miner.
```

Comment:
OK, so this is the [P2P network protocol](https://bitcoin.org/en/developer-reference#p2p-network)? Or is it the [get block template RPC protocol](https://bitcoin.org/en/developer-guide#getblocktemplate-rpc)? Either way, describe in the connection management section (see above) what the minimal thing is that the client must be running, and give a reference to its specification/documentation.